### PR TITLE
Create gnucobol-3.0-rc1-GCC-6.2.0.eb

### PR DESCRIPTION
**Pull request by levlafayette**
GNUCobol will be a useful addition to the suite.

### CODE REVIEW

File: easybuild/easyconfigs/g/GNUCobol/gnucobol-3.0-rc1-GCC-6.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Lev Lafayette, University of Melbourne, 20190808
+easyblock = 'ConfigureMake'
+
+name = 'gnucobol'
+version = '3.0-rc1'
+
+homepage = 'https://sourceforge.net/projects/open-cobol/'
+description = """GnuCOBOL (formerly OpenCOBOL) is a free COBOL compiler. cobc translates COBOL source to executable using intermediate C, designated C compiler and linker."""
+
+toolchain = {'name': 'GCC', 'version': '6.2.0'}
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/g/GNUCobol/gnucobol-3.0-rc1-GCC-6.2.0.eb]:
```suggestion
toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
```

and rename file accordingly?

File: easybuild/easyconfigs/g/GNUCobol/gnucobol-3.0-rc1-GCC-6.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+sources = [SOURCELOWER_TAR_XZ]
+source_urls = ['http://downloads.sourceforge.net/open-cobol/gnucobol-3.0-rc1.tar.xz']
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/g/GNUCobol/gnucobol-3.0-rc1-GCC-6.2.0.eb]:
```suggestion
source_urls = ['http://downloads.sourceforge.net/open-cobol/']
```

File: easybuild/easyconfigs/g/GNUCobol/gnucobol-3.0-rc1-GCC-6.2.0.eb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+dependencies = [
+        ('DB', '6.2.32'),
+        ('GMP', '6.1.1'),
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/g/GNUCobol/gnucobol-3.0-rc1-GCC-6.2.0.eb]:
```suggestion
        ('GMP', '6.1.2'),
```
to match the existing version for this toolchain

File: easybuild/easyconfigs/g/GNUCobol/gnucobol-3.0-rc1-GCC-6.2.0.eb [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+name = 'gnucobol'
+version = '3.0-rc1'
+
+homepage = 'https://sourceforge.net/projects/open-cobol/'
+description = """GnuCOBOL (formerly OpenCOBOL) is a free COBOL compiler. cobc translates COBOL source to executable using intermediate C, designated C compiler and linker."""
+
+toolchain = {'name': 'GCC', 'version': '6.2.0'}
+
+sources = [SOURCELOWER_TAR_XZ]
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/g/GNUCobol/gnucobol-3.0-rc1-GCC-6.2.0.eb]: missing checksum, you can add it with `--inject-checksums`

File: easybuild/easyconfigs/g/GNUCobol/gnucobol-3.0-rc1-GCC-6.2.0.eb [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# Lev Lafayette, University of Melbourne, 20190808
+easyblock = 'ConfigureMake'
+
+name = 'gnucobol'
+version = '3.0-rc1'
+
+homepage = 'https://sourceforge.net/projects/open-cobol/'
+description = """GnuCOBOL (formerly OpenCOBOL) is a free COBOL compiler. cobc translates COBOL source to executable using intermediate C, designated C compiler and linker."""
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/g/GNUCobol/gnucobol-3.0-rc1-GCC-6.2.0.eb]: line too long

File: easybuild/easyconfigs/g/GNUCobol/gnucobol-3.0-rc1-GCC-6.2.0.eb [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+sources = [SOURCELOWER_TAR_XZ]
+source_urls = ['http://downloads.sourceforge.net/open-cobol/gnucobol-3.0-rc1.tar.xz']
+
+dependencies = [
+        ('DB', '6.2.32'),
+        ('GMP', '6.1.1'),
+        ('libtool', '2.4.6'),
+        ('ncurses', '6.0'),
+        ]
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/g/GNUCobol/gnucobol-3.0-rc1-GCC-6.2.0.eb]: this block is not indented correctly (you can check with `--check-contrib`)

**Comment by boegel** [comment]
No updates in over 2 years, so I'll go ahead and close this PR...

**Pull request closed without merging**